src/panels/lovelace/editor/select-view/hui-dialog-select-view.ts

```diff
@@ -94,12 +96,43 @@ export class HuiDialogSelectView extends LitElement {
             </ha-paper-dropdown-menu>`
           : ""}
         ${this._config
-          ? html` <hui-views-list
-              .lovelaceConfig=${this._config}
-              @view-selected=${this._selectView}
-            >
-            </hui-views-list>`
+          ? html`
+              <ha-paper-dropdown-menu
+                .label=${this.hass.localize(
+                  "ui.panel.lovelace.editor.select_view.views_label"
+                )}
+              >
+                <paper-listbox
+                  slot="dropdown-content"
+                  .selected=${this._selectedViewIdx}
+                >
+                  ${this._config.views.map(
+                    (view, idx) => html`
+                      <paper-icon-item
```

Comment:
We should only reserve space for an icon if any of the views have an icon, check how `hui-views-list` did this